src/NUnitFramework/framework/Internal/Commands/OneTimeTearDownCommand.cs

```diff
@@ -22,19 +20,7 @@ public OneTimeTearDownCommand(TestCommand innerCommand, SetUpTearDownItem setUpT
             Guard.ArgumentValid(innerCommand.Test is TestSuite, "OneTimeTearDownCommand may only apply to a TestSuite", nameof(innerCommand));
             Guard.ArgumentNotNull(setUpTearDownItem, nameof(setUpTearDownItem));
 
-            AfterTest = context =>
-            {
-                TestResult suiteResult = context.CurrentResult;
-
-                try
-                {
-                    setUpTearDownItem.RunTearDown(context);
-                }
-                catch (Exception ex)
-                {
-                    suiteResult.RecordTearDownException(ex);
```

Comment:
Why should not this be run?

Reply:
Because the exceptions are already caught in the `RunTearDown` method hence it would never get into the `catch` part.